app/code/Magento/Downloadable/Console/Command/DomainsRemoveCommand.php

```diff
@@ -30,15 +32,23 @@ class DomainsRemoveCommand extends Command
      */
     private $domainManager;
 
+    /**
+     * @var DownloadManager
+     */
+    private $downloadManager;
+
     /**
      * DomainsRemoveCommand constructor.
      *
      * @param DomainManager $domainManager
+     * @param DownloadManager $downloadManager
      */
     public function __construct(
-        DomainManager $domainManager
+        DomainManager   $domainManager,
+        DownloadManager $downloadManager
```

Comment:
Adding new parameter in constructor is BiC change, requesting you to mark it as `null` and use ObjectManager to initialise it.